tests/unit/h/conftest.py

```diff
@@ -207,7 +140,7 @@ def pyramid_config(pyramid_settings, pyramid_request):
 def pyramid_request(db_session, fake_feature, pyramid_settings):
     """Return pyramid request object."""
     request = testing.DummyRequest(db=db_session, feature=fake_feature)
-    request.default_authority = TEST_AUTHORITY
+    request.default_authority = "example.com"
```

Comment:
This only appears in the file once now so I replaced the variable with a literal